### PR TITLE
[MNGSITE-397] Rewrite and clarify

### PR DESCRIPTION
**Pull request by elharo**
@EPadronU

### CODE REVIEW

File: content/markdown/what-is-maven.md [hunk 1]
```diff
@@ -35,52 +35,44 @@ with the comprehension of any Java-based project.
...
-to attain this goal, there are several areas of concern that Maven
-attempts to deal with:
+to attain this goal, Maven deals with several areas of concern:
 
 -   Making the build process easy
 -   Providing a uniform build system
 -   Providing quality project information
--   Providing guidelines for best practices development
--   Allowing transparent migration to new features
+-   Encouraging best practices development
```
Comment (EPadronU) [inline content/markdown/what-is-maven.md]: How about `Encouraging best development practices`?

**Comment by EPadronU** [comment]
How may I assist you?

File: content/markdown/what-is-maven.md (hunk 1 above)
Comment (elharo) [reply content/markdown/what-is-maven.md]: Good idea. Changed to "Encouraging better development practices"

**Comment by EPadronU** [comment]
It looks good to me.

**Pull request merged**